packages/microsoft-reactnative-sampleapps/codegen/NativeMyModuleSpec.g.h

```diff
@@ -19,10 +19,10 @@ struct MyModuleSpec : winrt::Microsoft::ReactNative::TurboModuleSpec {
       SyncMethod<bool(bool) noexcept>{1, L"getBool"},
       SyncMethod<double(double) noexcept>{2, L"getNumber"},
       SyncMethod<std::string(std::string) noexcept>{3, L"getString"},
-      SyncMethod<JSValueArray(JSValueArray) noexcept>{4, L"getArray"},
-      SyncMethod<JSValueObject(JSValueObject) noexcept>{5, L"getObject"},
-      SyncMethod<JSValueObject(double, std::string, JSValueObject) noexcept>{6, L"getValue"},
-      Method<void(Callback<JSValue>) noexcept>{7, L"getValueWithCallback"},
+      SyncMethod<React::JSValueArray(React::JSValueArray) noexcept>{4, L"getArray"},
```

Comment:
Just for my sanity: 
I didn't see any change to something that looks like a code generator, nor a package version update in case the codegen is part of that... Did you manually change this?

Reply:
No, I did not. This file is updated every time I run 'yarn build'. The change is related to the recent codegen changes by Andrew Coates.

---
In reply to: [430597854](https://github.com/microsoft/react-native-windows/pull/5007#discussion_r430597854) [](ancestors = 430597854)